src/tests/ftest/util/test_utils_pool.py

```diff
@@ -91,10 +102,14 @@ def create(self):
             self.log.info("Creating a pool")
 
         self.pool = DaosPool(self.context)
+
+        self.label.update(self.label_generator.get_label(self.label.value))
```

Comment:
We should provide the option to still create a pool w/o a label, but if we do create a pool with a label ensure that the `LabelGenerator` is defined:
```suggestion 
        # Use a unique pool label if using pool labels
        if self.label.value is not None:
            if not isinstance(self.label_generator, LabelGenerator):
                raise CommandFailure(
                    "Unable to create a unique pool label; Undefined label_generator")
            self.label.update(self.label_generator.get_label(self.label.value))
```